src/types/Order.ts

```diff
@@ -0,0 +1,346 @@
+import { ContactMech, Enumeration, Party, Status, Uom } from "./index"
+
+export interface Order {
+  orderId: string,
+  orderName?: string,
+  entryDate?: string,
+  placedDate?: string,
+  approvedDate?: string,
+  completedDate?: string,
+  statusId?: string,
+  processingStatusId?: string,
+  orderRevision?: number,
+  currencyUomId?: string,
+  billingAccountId?: string,
+  productStoreId?: string,
+  salesChannelEnumId?: string,
+  externalId?: string,
+  externalRevision?: string,
+  originId?: string,
+  originUrl?: string,
+  syncStatusId?: string, 
+  systemMessageRemoteId?: string,
+  enteredByPartyId?: string,
+  parentOrderId?: string,
+  lastOrderedDate?: string,
+  recurAutoInvoice?: string,
+  remainingSubTotal?: number,
+  grandTotal?: number,
+  status?: Status,
+  currencyUom?: Uom,
+  salesChannel?: Enumeration,
+  parts?: Array<OrderPart>,
+  notes?: Array<{
```

Comment:
In current version, We decided to use communicationEvents for Notes. 

Should we follow same pattern?

Reply:
Had a discussion with @dixitdeepak Sir on the pattern to be followed here, so concluded that we will be using commnicationEvents for storing the Notes and will be removing notes from the schema.